pkg/formula/creator/template/default.go

```diff
@@ -19,8 +19,13 @@ package template
 const (
 	HelpJson = `
 {
-    "short": "short help placeholder for {{folderName}}",
-    "long": "long help placeholder for {{folderName}} used by index page and -h"
+    "short": "{{command}}commands",
+    "long": "{{command}}commands"
+}`
+	FormulaHelpJson = `
+{
+    "short": "{{command}}formula",
```

Comment:
If you already add a space on the template you will be able to  join the commands more elegantly
```suggestion
    "short": "{{command}} formula",
```

Reply:
yes! done in a68d9d602086fa25241f8b7ee7ddd9fe28d4e853